daprdocs/content/en/go-sdk-docs/go-client/_index.md

```diff
@@ -145,10 +145,6 @@ err := testClient.ExecuteStateTransaction(ctx, store, meta, ops)
 For a full guide on state management, visit [How-To: Save & get state]({{< ref howto-get-save-state.md >}}).
 
 
-### Query state
-
```

Comment:
Please add TODO not remove it directly.

Reply:
@daixiang0 Until we have content for that section, I'd rather remove completely from the docs - unless you have a go-sdk example for query state that I can use here?